### ISSUE TITLE
New Subtask won't save

### ISSUE THREAD

**Issue by rainbowjaw** [wekan/wekan#1916]
**Server Setup Information**:
* Wekan version:  **1.49.0~2018-09-17**
* Operating System:  **Windows 10**
* Deployment Method: **Sandstorm**

**Problem description**:

Can't create a new subtask. When naming a new subtask, it won't save. 
Pressing the "save" button grey's out text box, but never creates a new subtask. Only way forward is to cancel. 
![subtask](https://user-images.githubusercontent.com/39452251/46091905-fd999980-c170-11e8-83cb-113c2cc0983a.GIF)

- *Explain steps how to reproduce*: 

Type in subtask in text field, press save. Nothing happens. 

- *In webbrowser, what does show Right Click / Inspect / Console ?:
```
<form class="inlined-form js-inlined-form js-add-subtask"><textarea class="js-add-subtask-item" rows="1" autofocus=""></textarea><div class="edit-controls clearfix"><button class="primary confirm js-submit-add-subtask-item-form" type="submit">Save</button>
<a class="fa fa-times-thin js-close-inlined-form"></a></div></form>
```

**Comment by rainbowjaw** [comment]
An Additional Test Case (same server, different account)
Mac OS (2017ish, sorry don't know the Version.   new mac book pro from last year)


Same results, subtask won't save.

**Comment by rainbowjaw** [comment]
Found a similar bug where "save" on a checklist wouldn't operate until i resolved a save/cancel function on a separate checklist on the same page.

**Comment by sfahrenholz** [comment]
I testing it under ubuntu with the snap installation and this works..
What i see, foreach subtasks there create a new list with the name "queue" and the card on it..

**Comment by sfahrenholz** [comment]
This is not only a Sandstorm bug. it is also available on v2.43 on snap version

**Comment by xet7** [comment]
## Workaround to get Subtasks working

https://github.com/wekan/wekan/wiki/Subtasks

Moved to #2256
